### PR TITLE
feat(purchases): update to plugin version 2.2.0

### PR DESCRIPTION
**Pull request by beylmk**
Updates to cordova-plugin-purchases 2.2.0

### CODE REVIEW

File: src/@ionic-native/plugins/purchases/index.ts [hunk 1]
```diff
@@ -290,7 +329,7 @@ export class Purchases extends IonicNativePlugin {
   /**
    * Gets the Offerings configured in the dashboard
    *
-   * @return {Observable<PurchasesOfferings>} Will return a [PurchasesError] if the offerings are not properly configured in RevenueCat or if there is another error retrieving them.
+   * @return {Promise<PurchasesOfferings>} Will return a [PurchasesError] if the offerings are not properly configured in RevenueCat or if there is another error retrieving them.
```
Comment (vegaro) [inline src/@ionic-native/plugins/purchases/index.ts:332]: good catch

File: src/@ionic-native/plugins/purchases/index.ts [hunk 2]
```diff
@@ -682,7 +737,7 @@ export class Purchases extends IonicNativePlugin {
    * @param onesignalID Empty String or null will delete the subscriber attribute.
    */
   @Cordova({ sync: true })
-  setOnesignalId(onesignalId: string | null): void {}
+  setOnesignalID(onesignalID: string | null): void {}
```
Comment (vegaro) [inline src/@ionic-native/plugins/purchases/index.ts:740]: These changes are going to break people's implementations. Did we change the names in cordova too?

Comment (beylmk) [reply src/@ionic-native/plugins/purchases/index.ts:740]: actually it looks like they've always been uppercase in cordova, so not sure how this ever worked...

see: https://github.com/RevenueCat/cordova-plugin-purchases/commit/c803df61a861b6e96b3ac24b465bfde80a106baf#diff-5776869548f9a7e95693a9bf21febcb3d6e3c808fc9ed4533444e03781400958

**Review by danielsogl — APPROVED** [review]

**Comment by danielsogl** [comment]
@beylmk please resolve the merge conflicts

**Comment by beylmk** [comment]
> @beylmk please resolve the merge conflicts

@danielsogl closing this, as a newer release has already been merged with all these changes

**Pull request closed without merging**